### PR TITLE
FYLE-85zrkx6jk Code coverage: Pending User Service method

### PR DESCRIPTION
**Pull request by CODEX108**

### CODE REVIEW

**Comment by CODEX108** [comment]
@jayfyle @Dimple16, the `method names` aren't mentioned in the spec file for other methods can I include those names too?
`<method name> should do this`

File: src/app/core/services/user.service.spec.ts [hunk 1]
```diff
@@ -181,6 +181,16 @@ const userPropertiesNew: UserProperty = {
   expense_form_beta: null,
 };
 
+const userPasswordStatus: UserPasswordStatus = {
```
Comment (Dimple16) [inline src/app/core/services/user.service.spec.ts:184]: add these 2 variables in a separate file -> `/mock-data/user-password-status.data.ts`

**Review by Dimple16 — CHANGES_REQUESTED** [review]

File: src/app/core/mock-data/user.data.ts [hunk 2]
```diff
@@ -0,0 +1,6 @@
+import { UserPasswordStatus } from '../models/user-password-status.model';
+
+export const userPasswordStatus: UserPasswordStatus = {
```
Comment (Dimple16) [inline src/app/core/mock-data/user.data.ts:3]: the name of this file should be `user-password-status.data.ts`

File: src/app/core/services/user.service.spec.ts [hunk 3]
```diff
@@ -262,4 +262,14 @@ describe('UserService', () => {
       done();
     });
   });
+
+  it('getUserPasswordStatus() :should get the user password status', (done) => {
+    apiService.get.and.returnValue(of(userPasswordStatus));
+    userService.getUserPasswordStatus().subscribe((res) => {
+      expect(userPasswordStatus).toEqual(res);
+      expect(apiService.get).toHaveBeenCalledWith('/users/password_required');
```
Comment (Dimple16) [inline src/app/core/services/user.service.spec.ts:270]: use `toHaveBeenCalledOnceWith` and remove the below line

**Review by Dimple16 — CHANGES_REQUESTED** [review]
added comments

**Review by Dimple16 — APPROVED** [review]

**Review by jayfyle — APPROVED** [review]

**Pull request merged**